tensorflow/python/keras/mixed_precision/experimental/autocast_variable_test.py

```diff
@@ -40,41 +43,31 @@
 from tensorflow.python.training import gradient_descent as gradient_descent_v1
 from tensorflow.python.training.tracking import util as trackable_utils
 
-TESTCASES = ({
-    'testcase_name': 'base',
-    'distribute': False
-}, {
-    'testcase_name': 'distribute',
-    'distribute': True
-})
-
-
-def get_distribute_scope(distribute):
-
-  class DummyContextManager(object):
-
-    def __enter__(self):
-      pass
-
-    def __exit__(self, *args):
-      pass
-
-  if distribute:
-    return mirrored_strategy.MirroredStrategy(['cpu:0']).scope()
-  else:
-    return DummyContextManager()
+class DummyStrategy(object):
```

Comment:
Better to use the default Strategy, as is done [here](https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/python/keras/mixed_precision/experimental/keras_test.py;l=96;drc=f9e99f4dca729f6c21155641103165b6dfac4730). Note the scope() function for the default strategy does nothing. Then change the `NamedDistribution` to use it and rename it from "Dummy" to "Default"

Reply:
Sounds good, I changed it to `strategy_combinations.default_strategy`. I didn't do this in the first place since I wasn's sure if this was done on purpose due to some behaviour of the default Strategy I wasn't aware, but this works well.